uisdk/src/main/res/layout/uisdk_booking_checkout_view.xml

```diff
@@ -64,16 +64,39 @@
                     android:layout_weight="1" />
             </LinearLayout>
 
-            <com.karhoo.uisdk.screen.booking.checkout.comment.BookingOptionalInfoView
-                android:id="@+id/bookingRequestFlightDetailsWidget"
+            <LinearLayout
```

Comment:
Again, why do we need a linear layout? Seems like the same effect can be achieved with adding one single margin modifier to the nested `ActionCellView`